### ISSUE TITLE
strange ports/python/numpy/Makefile

### ISSUE THREAD

**Issue by lxliangit** [PacificBiosciences/pitchfork#270]
Does anyone find that is wired?

do-install: | $(PREFIX)/var/pkg/$(_NAME)
$(PREFIX)/var/pkg/$(_NAME):
	rm -rf $(STAGING)/$(_NAME)
	mkdir -p $(STAGING)/$(_NAME)
	$(PIP) install --root $(STAGING)/$(_NAME) --no-deps $(_WRKSRC)/
	rsync -aKx $(STAGING)/$(_NAME)/$(PREFIX)/ $(PREFIX)/
cd $(STAGING)/$(_NAME)$(PREFIX) && find * |grep -v '^$$'|sort -r > $@

The "rsync" command and following "cd" command seems wired. If I am not wrong, $PREFIX should be a path which is specified in settings.mk, so, what exactly these two commands suppose to do?

Thank you very much~

**Comment by mhsieh** [comment]
The purpose for the last line was write list of all installed files of numpyt to the text file at $PREFIX/var/pkg/numpy-x.x.x

**Comment by lxliangit** [comment]
Thank you very much.

I asked this question because I encounter some errors when install GenomicConsensus. I have figured out the reason, it's because I used some existing numpy instead of install it by pitchfork (by changing the Makefile to remove numpy dependency and some other symlink tricky steps). The reason I used the existing numpy (and pysam) is because I am playing around on the server on which I didn't have admin access, and I have some difficulties to install them, and blablabla...

Anyway, the design of pitchfork is really cool, thank you very much for that.
